collections/nemo_nlp/nemo_nlp/data/datasets/utils.py

```diff
@@ -547,6 +547,134 @@ def merge(data_dir, subdirs, dataset_name, modes=['train', 'test']):
     return outfold, none_slot
 
 
+def get_intent_query_files(path):
+    files = []
+    # r=root, d=directories, f = files
+    for r, d, f in os.walk(path):
```

Comment:
can we name the vars directly as root, dir_, file instead of r, d, f?

Reply:
Done.